dev/VSIX/ProjectTemplates/Desktop/CSharp/PackagedApp/BlankApp/MainWindow.xaml.cs

```diff
@@ -16,21 +16,20 @@
 // To learn more about WinUI, the WinUI project structure,
 // and more about our project templates, see: http://aka.ms/winui-project-info.
 
-namespace $safeprojectname$
+namespace $safeprojectname$;
+
+/// <summary>
+/// An empty window that can be used on its own or navigated to within a Frame.
+/// </summary>
+public sealed partial class MainWindow : Window
 {
-    /// <summary>
-    /// An empty window that can be used on its own or navigated to within a Frame.
-    /// </summary>
-    public sealed partial class MainWindow : Window
+    public MainWindow()
     {
-        public MainWindow()
-        {
-            this.InitializeComponent();
-        }
+        this.InitializeComponent();
+    }
 
-        private void myButton_Click(object sender, RoutedEventArgs e)
-        {
-            myButton.Content = "Clicked";
-        }
+    private void myButton_Click(object sender, RoutedEventArgs e)
+    {
+        myButton.Content = "Clicked";
     }
```

Comment:
While we're here, can we align the MainWindow here to match the BlankWindow ItemTemplate and just be an empty Grid without the Button and handler here? Thanks!

Reply:
Done, the MainWindow only have empty Grid now

Reply:
I don't agree with this change. Without this, the default app template will draw nothing. I find the simple button useful as something to confirm the new app is working, useful as a simple example for someone not familiar with XAML, and useful as an example to start adding more content.

Reply:
Personally agree with Michael's opinion - when i was ramping up for the WinAppSDK as a new developer in this area, a simple clickable component can directly show me how the app is working. @michael-hawker and @niels9001, how do you think @codendone's idea about still keeping the button? 
From the code history, the BlankWindow ItemTemplate (which have only an empty Grid without the Button) seems to be created by @Scottj1s, would like to know your preference as well.

Reply:
Here's my $0.02s:

- I often create a new project, change the XAML, hit F5 and get an error because I forgot to remove the C# code as well.  It's just a small pain point I run into every. single. time. 😁.
- The title of the template is "Blank app". But then it really isn't, right? "Blank app" for WPF and UWP means just that: completely blank.
- I don't think a `StackPanel` would ever be used as a root element of a `Window`? It would probably be a `Grid` to provide as much flexibility as possible (and is inline with WPF/UWP templates).
- We've heard a lot of feedback from developers that they would like to see the Button+TextBlock go, because of the above reasons.
- The main goal of a template in my mind is that it should make me productive: provide all of the basic scaffolding I need to get going (and as a Blank app, try to limit the amount of code as much as possible. Education can be a 2nd goal in case it does not interfere with the first goal.

I could imagine having only a `Button` (with Horizontal/Vertical alignment set to center) wrapped in a `Grid` would at least solve the `StackPanel` problem. For people new to XAML I'd expect [our getting started guides](https://learn.microsoft.com/en-us/windows/apps/how-tos/hello-world-winui3#display-hello-world) should help them better understand how to use XAML and get started?

Reply:
> * The main goal of a template in my mind is that it should make me productive: provide all of the basic scaffolding I need to get going (and as a Blank app, try to limit the amount of code as much as possible. Education can be a 2nd goal in case it does not interfere with the first goal.

Quite make sense to me, I may have previously missed the intended purpose of our template. Thanks for the insights Michael and Niels!